stories/examples/iris-dataset.stories.tsx

```diff
@@ -4,7 +4,8 @@ import { PCA as MlPCA } from 'ml-pca';
 import LinearRegression from 'ml-regression-simple-linear';
 import { ReactElement } from 'react';
 
-import { LineSeries, Plot, ScatterSeries, SeriesPoint, Axis } from '../../src';
+import { Plot, ScatterSeries, SeriesPoint, Axis } from '../../src';
+import { FunctionSeries } from '../../src/components/FunctionSeries';
```

Comment:
The new component should be exported like the others from `src/index.ts`, otherwise it is not possible to use it from another project.
Please also update this line and in the other story to import the component from src.

Reply:
fixed